iceberg/iceberg-rest-server/src/test/java/org/apache/gravitino/iceberg/service/metrics/TestIcebergMetricsFormatter.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.gravitino.iceberg.service.metrics;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import org.apache.iceberg.metrics.ImmutableCommitMetricsResult;
+import org.apache.iceberg.metrics.ImmutableCommitReport;
+import org.apache.iceberg.metrics.MetricsReport;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class TestIcebergMetricsFormatter {
+
+  private static MetricsReport createMetricsReport() {
+    ImmutableCommitMetricsResult commitMetricsResult =
+        ImmutableCommitMetricsResult.builder().build();
+    return ImmutableCommitReport.builder()
+        .tableName("a")
+        .snapshotId(1)
+        .sequenceNumber(1)
+        .operation("select")
+        .commitMetrics(commitMetricsResult)
+        .build();
+  }
+
+  private static IcebergMetricsFormatter formatter;
+  private static MetricsReport report;
+  private static final String expectedString =
+      "{\"table-name\":\"a\",\"snapshot-id\":1,\"sequence-number\":1,\"operation\":\"select\","
```

Comment:
the test result relies on the content of `ImmutableCommitMetricsResult` which is out of our control and hard to maintain if the upstream adds some fields to it.

Reply:
Perhaps just check for the values we set?

Reply:
yes, this is an acceptable solution.

Reply:
Good point. I posted an update to the PR.

Reply:
Good point. I posted an update.

Reply:
LGTM, thanks for your contribution